src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -30,11 +31,55 @@ function InboxReason({inbox, fontSize = 'sm'}: Props) {
       ? `More than ${Math.round(count / EVENT_ROUND_LIMIT)}k`
       : `${count}`;
 
+  function getTooltipDescription() {
+    const {until, count, window, user_count, user_window} = reason_details;
+    if (until) {
+      // Was ignored until `until` has passed.
+      //`until` format: "2021-01-20T03:59:03+00:00"
+      return tct('Was ignored until [window]', {
+        window: <DateTime date={until} dateOnly />,
+      });
+    }
+
+    if (count) {
+      // Was ignored until `count` events occurred
+      // If `window` is defined, than `count` events occurred in `window` minutes.
+      // else `count` events occurred since it was ignored.
+      if (window) {
+        return tct('Was ignored until it occurred [count] time(s) in [duration]', {
+          count: getCountText(count),
+          duration: getDuration(window * 60, 0, true),
+        });
+      }
+
+      return tct('Was ignored until it occurred [count] time(s)', {
+        count: getCountText(count),
+      });
+    }
+
+    if (user_count) {
+      // Was ignored until `user_count` users were affected
+      // If `user_window` is defined, than `user_count` users affected in `user_window` minutes.
+      // else `user_count` events occurred since it was ignored.
+      if (user_window) {
+        return t('Was ignored until it affected [count] user(s) in [duration]', {
+          count: getCountText(user_count),
+          duration: getDuration(user_window * 60, 0, true),
+        });
+      }
+      return t('Was ignored until it affected [count] user(s)', {
+        count: getCountText(user_count),
+      });
+    }
+
+    return undefined;
+  }
+
   function getReasonDetails(): {
     tagType: React.ComponentProps<typeof Tag>['type'];
     reasonBadgeText: string;
     tooltipText?: string;
-    tooltipDescription?: string;
+    tooltipDescription?: string | React.ReactNode;
```

Comment:
I would use `React.ReactElement `instead of `React.ReactNode `or only `React.ReactNode`, because it already contains the type `string` 😉

Reply:
Changed it :-)